plugins/openstack-common/src/main/java/org/eclipse/xpanse/plugins/openstack/common/auth/keystone/OpenstackKeystoneManager.java

```diff
@@ -121,14 +121,40 @@ public OSClient.OSClientV3 getAuthenticatedClient(String authUrl,
                 .authenticate();
     }
 
-    private Config buildClientConfig(String url, String proxyHost, String proxyPort,
-                                     String sslDisabled) {
-        Config config = Config.newConfig()
-                .withEndpointNATResolution(getIpAddressFromUrl(url))
-                .withEndpointURLResolver(new CustomEndPointResolver())
-                .withSSLVerificationDisabled()
-                .withProxy(Objects.nonNull(proxyHost)
-                        ? ProxyHost.of(proxyHost, Integer.parseInt(proxyPort)) : null);
+    private Config buildClientConfig(String url, String sslDisabled) {
+        Config config = null;
+        if (proxyConfigurationManager.isSystemProxyConfigured()) {
```

Comment:
HuaweiCloudSDK also supports configuring proxy in httpConfig as following:
        httpConfig.withProxyHost("proxy.huaweicloud.com")
                .withProxyPort(8080)
                .withProxyUsername(System.getenv("PROXY_USERNAME"))
                .withProxyPassword(System.getenv("PROXY_PASSWORD"));